src/btree/bt_random.c

```diff
@@ -302,11 +305,15 @@ __wt_btcur_next_random(WT_CURSOR_BTREE *cbt)
 	WT_UPDATE *upd;
 	wt_off_t size;
 	uint64_t n, skip;
+	uint32_t descent_flags;
```

Comment:
Was unsure if I should just keep this as simple `flags` or because this was only being forwarded into the descent method, a little more descriptive.

Reply:
The more descriptive name is helpful since it is only sent to descent.